procinterrupts.c

```diff
@@ -214,6 +244,24 @@ GList* collect_full_irq_list()
 		*c = 0;
 		number = strtoul(line, NULL, 10);
 
+#ifdef AARCH64
+		if (is_arm_irq_aff_cannot_change(number)) {
+			/*
+			 * This means that the irq affinity cannot be changed, just like:
+			 * (1) the irq with IRQF_PERCPU flag,  per cpu irq (in arm64, like PPI)
+			 * (2) the irq with IRQD_NO_BALANCING flag, some driver request irq can
+			 *     set the flag according to themselves require. for example in arm64,
+			 *     for the passthrough doorbell irq (GICV4), in future.
+			 * (3) the irq with IRQD_AFFINITY_MANAGED flag, some drivers can set
+			 *     specially irq affinity, and prohibit user to modify it.
+			 *
+			 * For these irqs, we can add these to banned irq list.
+			 */
+			add_banned_list_irq(number);
```

Comment:
Why not just use add_banned_irq here?

Reply:
banned_irqs and add_banned_irq is static in classify.c

Reply:
see comment above, theres a better way to do this

Reply:
fixed